target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -1,21 +1,28 @@
-# Copyright (C) 2014 OpenWrt.org
+#!/bin/sh
```

Comment:
No shebang in lib files. And please don't remove any Copyright marks.

Reply:
thanks, got rid of shebang and brought back copyright mark